OtripleS.Web.Api/Controllers/ContactsController.cs

```diff
@@ -0,0 +1,64 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using OtripleS.Web.Api.Models.Contacts;
+using OtripleS.Web.Api.Models.Contacts.Exceptions;
+using OtripleS.Web.Api.Services.Contacts;
+using RESTFulSense.Controllers;
+
+namespace OtripleS.Web.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ContactsController : RESTFulController
+    {
+        private readonly IContactService contactService;
+
+        public ContactsController(IContactService contactService) => this.contactService = contactService;
+
+        [HttpPost]
+        public async ValueTask<ActionResult<Contact>> PostContactAsync(Contact contact)
```

Comment:
@viralpandya  can you please check, its throwing which doesn't belongs to this controller and service. 

![error](https://user-images.githubusercontent.com/5396746/94232419-31a02480-ff1b-11ea-8e7d-9daa900ed1e9.jpg)